python/astro_metadata_translator/tests.py

```diff
@@ -124,9 +125,10 @@ def assertCoordinatesConsistent(self, obsinfo, max_sep=1.0, amdelta=0.01):  # no
             sep = obsinfo.altaz_begin.separation(obsinfo.tracking_radec.altaz)
         self.assertLess(sep.to_value(unit="arcmin"), max_sep, msg="AltAz inconsistent with RA/Dec")
 
-    def assertObservationInfoFromYaml(self, file, dir=None, check_wcs=True,  # noqa: N802
-                                      wcs_params=None, **kwargs):
-        """Check contents of an ObservationInfo.
+    def assertObservationInfoConsistent(self, file, dir=None, check_wcs=True,  # noqa: N802
```

Comment:
I'm not entirely sure we need to change the method name do we? The existing name is still true (although you are right that the "Yaml"-ness is not overly relevant, it's just a file.